docs/reference/migration/migrate_5_0/fs.asciidoc

```diff
@@ -8,3 +8,12 @@ there is nothing to worry about since this is only address space consumption
 and the actual memory usage of Elasticsearch will stay similar to what it was
 in 2.x. See http://blog.thetaphi.de/2012/07/use-lucenes-mmapdirectory-on-64bit.html
 for more information.
+
+=== Path to data on disk
+
+In prior versions of Elasticsearch, the `path.data` directory included a folder
+for the cluster name, so that data was in a folder such as
+`$DATA_DIR/$CLUSTER_NAME/nodes/$nodeOrdinal`. In 5.0 the cluster name is no
+longer used as a folder name, so the data is in `$DATA_DIR/nodes/$nodeOrdinal`.
+Upon startup, Elasticsearch will automatically upgrade directories to the new
+folder structure.
```

Comment:
I guess we should also let folks know if they have multi cluster systems they can add the clustername to the datapath they configured?